SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md

```diff
@@ -0,0 +1,75 @@
+<%(TOC)%>
+# Multiserver Shooter Tutorial
+
+## Step 4: Test your changes in the cloud
+
+### Build your assemblies
+
+An assembly is what’s created when you run `BuildWorker.bat`. They’re .zip files that contains all the files that your game uses when running in the cloud.
```

Comment:
```suggestion
An assembly is what’s created when you run `BuildWorker.bat`. They’re .zip files that contain all the files that your game uses when running in the cloud.
```

Reply:
I think we can use the same text as in the new ones here: https://docs.google.com/document/d/1AwEVa4H-IN4pwWgbs6Yn0CjSGma6IyCXkiq61i5MvDE/edit (Subject to Mike S approval)